SwrveSDK/SDK/Track/Swrve.m

```diff
@@ -1915,6 +1939,16 @@ - (void) queueDeviceProperties
     [self userUpdate:self.deviceInfo];
 }
 
+- (BOOL) canSendExtraLogs {
+    return self->extraLogs;
+}
+
+- (void) extraLogEvent:(NSString*)name payload:(NSDictionary*)payload {
+    if (self->extraLogs) {
+        [self eventInternal:[@"Swrve.extra_log." stringByAppendingString:name] payload:payload triggerCallback:NO];
```

Comment:
Similar to the comment on https://github.com/Swrve/swrve/pull/25526, I think we need a more specific name than 'extra_log' to future-proof ourselves.. possibly 'Swrve.debug.ios_campaign_state_log'?